module/observations/helper.js

```diff
@@ -178,12 +178,13 @@ module.exports = class ObservationsHelper {
     static createObservation(data,userId,solution,userRoleInformation="") {
         return new Promise(async (resolve, reject) => {
             try {
+    
                 if (data.entities) {
                     let entitiesToAdd = 
-                    await entitiesHelper.validateEntities(data.entities, solution.entityTypeId);
+                    await entitiesHelper.validateEntities(data.entities, solution.entityType);
```

Comment:
@VISHNUDAS-tunerlabs make sure in createObservation function call, inside solution entityType is passing everywhere

Reply:
checked